web/packages/teleterm/src/mainProcess/profileWatcher/profileWatcher.test.ts

```diff
@@ -49,6 +49,8 @@ afterEach(() => {
   abortController.abort();
 });
 
+const testDebounceMs = 100;
```

Comment:
I was going to suggest going as low as `10`, but "file system events are debounced and no events are lost when handler is slow" starts being flaky with that value.

Waiting just slightly longer than some interval can be quite flaky but I'm also guilty of doing it since it's often hard to implement this any other way. But maybe we could lower it to 10ms in other tests and use a higher value, such as 200, in the test that simulates a slow handler?